src/Equinor.ProCoSys.IPO.Query/PersonMinimalDto.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Equinor.ProCoSys.IPO.Query
+{
+    public class PersonMinimalDto
```

Comment:
Move to namespace where used, and rename to PersonDto

Reply:
Should I just use the same PersonDto everywhere where there is a Person reference? So it would be here, invitation.CreatedBy, SignedBy, AcceptedBy, CompeltedBy, Attachment.UploadedBy, etc.
There is already one, but it has rowVersion and azureOid (and email) on it. Should I just use this, alternatively see if I can remove rowVersion and Oid? (I dont see rowVersion and oid on PersonDto in preservation)